src/decrypt-api/decrypt-api.csproj

```diff
@@ -17,15 +17,16 @@
     <PackageReference Include="App.Metrics.AspNetCore.Mvc" Version="2.0.0" />
     <PackageReference Include="App.Metrics.Formatters.Prometheus" Version="2.0.0" />
     <PackageReference Include="KubernetesClient" Version="1.2.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.5" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.2.1" />
     <PackageReference Include="Serilog.AspNetCore" Version="2.1.1" />
+    <PackageReference Include="Microsoft.NETCore.App" Version="2.2.1" />
```

Comment:
AFAIK `Microsoft.NETCore.App` is not required, `Microsoft.AspNetCore.App` is enough.
for `Microsoft.AspNetCore.App` you shouldn't specify version since .netcore 2.2.

This is true for both encryptor and decryptor.

Reply:
Talking with Snyk about it